core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/FPCSchedulerServer.scala

```diff
@@ -79,7 +85,7 @@ object FPCSchedulerServer {
 
   private val schedulerUsername = loadConfigOrThrow[String](ConfigKeys.whiskSchedulerUsername)
   private val schedulerPassword = loadConfigOrThrow[String](ConfigKeys.whiskSchedulerPassword)
-  private val queuePathPrefix = "queue"
+  private val queuePathPrefix = "queues"
```

Comment:
The scheduler deployment no longer uses the number of queues.
It uses the number activations instead.
https://github.com/apache/openwhisk/pull/5338/files#diff-168a0313e0b46601cf796cd58db3a422a426015b91de821de1951bc88c62db7cR297

Reply:
When we deploy this feature first time, there has no `activation/count` API in old component, so the request against old component would be failed, on this case, what will happen?

Reply:
I got your point. This is a fair point.
Since the zero downtime deployment was not supported in the upstream version, I think there would be no issue with it.

It only matters in our downstream. In our downstream, I think we need some manual operations.
Since all APIs for each component are different from our downstream, I think we need to manually disable each component and wait until they are ready before deploying a component.